### ISSUE TITLE
VoiceOver does not announce the comparision operators in the query editor in a usable way

### ISSUE THREAD

**Issue by MRayermannMSFT** [Microsoft/AzureStorageExplorer#207]
VoiceOver is currently literally announcing the characters in the comparison operators (< > >= <= ==) for the query builder. It should instead be saying things like "greater than", "less than", etc.

Closing this will fix:
- https://devdiv.visualstudio.com/DevDiv/Storage%20Tools/_workitems/edit/604921

**Comment by craxal** [comment]
I have a fix ready. VoiceOver still doesn't work perfectly when the select element is expanded. For example, even though "<>" has an aria-label of "not equal", VoiceOver will pronounce this as "less than greater than" when reading from the expanded select element.
